.github/actions/upgrade-indexer/common.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+FILES_OLD="/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig"
+FILES_NEW="/etc/wazuh-indexer/opensearch-security"
+declare -A files_old
+declare -A files_new
+PACKAGE_NAME="$1"
```

Comment:
Use ${} for the variables, please check the rest of the script.

Reply:
Done.